src/domain/synthetics/trade/useSidecarOrdersState.ts

```diff
@@ -9,6 +9,23 @@ export function useSidecarOrdersState() {
   const [tpEntries, setTpEntries] = useState<SidecarOrderEntry[]>([]);
   const [limitEntries, setLimitEntries] = useState<SidecarOrderEntry[]>([]);
 
+  const [slEntriesPristine, setSlEntriesPristine] = useState<boolean>(true);
+  const [tpEntriesPristine, setTpEntriesPristine] = useState<boolean>(true);
+  const [limitEntriesPristine, setLimitEntriesPristine] = useState<boolean>(true);
```

Comment:
I tried to understand what you mean by pristine in ur PR but had no luck.
do you use it to reset entries? 
anyway let's come up with different name please?

Reply:
It's just a sign that state wasn't changed yet, just a shortcut for `!isChanged`. Same naming I used [here in this PR](https://github.com/gmx-io/gmx-interface/pull/1033). Seems pretty common naming though, saw it in react-final-form for example.